tests/notification/base.php

```diff
@@ -66,6 +66,8 @@ protected function setUp()
 			$phpbb_root_path,
 			$phpEx
 		);
+		
+		$phpbb_dispatcher = $this->phpbb_dispatcher = new phpbb_mock_event_dispatcher();
```

Comment:
The others are needed due to `global $db, $config, $user, $auth, $cache, $phpbb_container;`